docker-compose/obkm/scripts/README.md

```diff
@@ -0,0 +1,78 @@
+# WSO2 Open Banking API Manager with Open Banking Key Manager
```

Comment:
@Akila94 shall we update the pattern name?

Reply:
Updated. Thanks.